app/script/auth/route.js

```diff
@@ -23,6 +23,7 @@ const stagingWebsite = 'https://wire-website-staging.zinfra.io';
 
 export default {
   CREATE_ACCOUNT: '/createaccount',
+  CREATE_PERSONAL_ACCOUNT: '/createpersonalaccount',
```

Comment:
I would rather rename `CREATE_ACCOUNT` to `CREATE_TEAM_ACCOUNT` and go with `CREATE_ACCOUNT` for the personal one to keep names a little shorter.